validator/sawtooth_validator/execution/processor_handlers.py

```diff
@@ -44,6 +44,15 @@ def handle(self, connection_id, message_content):
         else:
             max_occupancy = request.max_occupancy
 
+        if request.request_header_style == \
+                processor_pb2.TpRegisterRequest.STYLE_UNSET:
+            header_style = processor_pb2.TpRegisterRequest.EXPANDED
+            LOGGER.debug('Transaction Processor with connection_id=%s request '
```

Comment:
This is probably too verbose and 99.99% of the time completely uninteresting. Suggest we not log anything here.